### PR TITLE
Add Bitergia login logo to the repository

### LINKED ISSUE #6: Wrong Bitergia logo URL
Bitergia's logo should be taken from this repository and not from kibiter one.

https://github.com/Bitergia/bitergia-analytics-odfe-kibana/blob/master/docker-entrypoint.sh#L106

### PR DESCRIPTION
**Pull request by dlumbrer**
This commit adds the Bitergia login logo to the repository because
the old one was in the Kibiter deprecated repository.

This should fix #6

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -103,7 +103,7 @@ fi
 if [ "$LOGIN_BRANDIMAGE" != "" ]; then
     printf "opendistro_security.basicauth.login.brandimage: \"$LOGIN_BRANDIMAGE\"\n" >> /opt/kibana/config/kibana.yml
 else
-    printf "opendistro_security.basicauth.login.brandimage: 'https://raw.githubusercontent.com/chaoss/grimoirelab-kibiter/integration-6.8.6/docker/assets/bitergia_login_logo.png'\n" >> /opt/kibana/config/kibana.yml
+    printf "opendistro_security.basicauth.login.brandimage: 'https://raw.githubusercontent.com/Bitergia/bitergia-analytics-odfe-kibana/logo_repo/assets/bitergia_login_logo.png'\n" >> /opt/kibana/config/kibana.yml
```
Comment (sduenas) [inline docker-entrypoint.sh]: The url has `logo_repo`. Shouldn't it be removed?

**Review by sduenas — CHANGES_REQUESTED** [review]

File: docker-entrypoint.sh (hunk 1 above)
Comment (dlumbrer) [reply docker-entrypoint.sh]: Yes, my bad, I changed it to "master"

**Comment by dlumbrer** [comment]
I updated the PR @sduenas , I forgot to change the branch name in the URL

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**